src/modules/uxrce_dds_client/uxrce_dds_client.cpp

```diff
@@ -694,8 +706,17 @@ UxrceddsClient *UxrceddsClient::instantiate(int argc, char *argv[])
 		}
 	}
 
+	// determines if timestamps should be synchronized
+	int32_t synchronize_timestamps = 0;
+	param_get(param_find("UXRCE_DDS_SYNCT"), &synchronize_timestamps);
+
+	if ((synchronize_timestamps != 1) && (synchronize_timestamps != 0)) {
+		PX4_ERR("UXRCE_DDS_SYNCT must be either 0 or 1");
```

Comment:
A simple PX4_INFO when starting if timestamp is enabled sounds fine.

Reply:
In addition to
https://github.com/PX4/PX4-Autopilot/blob/929f7e731d75f0de805bdf55bcd3744f2d5cf1a2/src/modules/uxrce_dds_client/uxrce_dds_client.cpp#L322-L323 ?
If synchronization is disabled, that info will no be displayed

No check if a invalid value is manually set?